lib/presentation/widgets/notification_header.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:sis_group6/bloc/tweets/tweets_bloc.dart';
+import 'package:sis_group6/controller/menu_controller.dart';
+import 'package:sis_group6/core/utils/responsive.dart';
+// import 'package:sis_group6/presentation/widgets/search_bar.dart';
+
+class Notification_Header extends StatelessWidget {
```

Comment:
Can remove this file and use `Header` widget instead

Reply:
removed header, done